dev/com.ibm.ws.springboot.support_fat/fat/src/com/ibm/ws/springboot/support/fat/Java11WithSpringBoot15Tests.java

```diff
@@ -0,0 +1,61 @@
+/*******************************************************************************
+ * Copyright (c) 2018 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.springboot.support.fat;
+
+import static componenttest.custom.junit.runner.Mode.TestMode.FULL;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.custom.junit.runner.Mode;
+
+@RunWith(FATRunner.class)
+@Mode(FULL)
+public class Java11WithSpringBoot15Tests extends AbstractSpringTests {
+    @Override
+    public boolean expectApplicationSuccess() {
+        return false;
+    }
+
+    @Override
+    public Set<String> getFeatures() {
+        return new HashSet<>(Arrays.asList("springBoot-1.5", "servlet-4.0"));
+    }
+
+    @Override
+    public String getApplication() {
+        return SPRING_BOOT_15_APP_BASE;
+    }
+
+    @Test
+    public void expectWarningMsgAndAppFailureWithJava11AndSpringBoot159() throws Exception {
+        String currentJavaVersion = System.getProperty("java.version");
+        if (currentJavaVersion.startsWith("1.")) {
+            server.stopServer();
+            //Update the java version to 11
+            System.setProperty("java.version", "11");
```

Comment:
How does this work, setting the java.version property here is for the FAT execution, that should not lead to the JVM used to launch the test server to think it is Java 11.  I think this test needs to be conditional to expect failure messages on Java 11 and success messages on Java 8.

Reply:
Refactored the test class.